kolibri/plugins/learn/assets/src/views/ExploreLibrariesPage/index.vue

```diff
@@ -127,7 +128,12 @@
         );
       },
       networkDevicesWithChannels() {
-        return this.networkDevices.filter(device => device.channels?.length > 0);
+        return this.networkDevices.filter(device => {
+          const connectionStatus = device['connection_status'];
+          const unallowedStatuses = [ConnectionStatus.Conflict, ConnectionStatus.ConnectionFailure];
+          const connectionIsOkay = !unallowedStatuses.includes(connectionStatus);
+          return device.channels?.length > 0 && connectionIsOkay;
+        });
```

Comment:
If `networkDevices` is retrieved from discovery's network location API, then it's perhaps simpler to use `available` which should exist on the response. The `available` attribute is passed through from the prop on the model:
https://github.com/learningequality/kolibri/blob/312e91287484c0f8313e7c8891915b4faa480457/kolibri/core/discovery/models.py#L111-L118

Reply:
Let me check this!

Reply:
Check has been updated to used the `available` attribute instead!